### PR TITLE
changed description field type to editor

### PR DESCRIPTION
**Pull request by JerryChr-zz**
I can't find any obvious/clear reason to why this field type is textarea so can't see the issue in changing it into editor. I've also encountered more times now when I needed it to be editor type.

### CODE REVIEW

File: administrator/components/com_banners/models/forms/banner.xml [hunk 1]
```diff
@@ -50,8 +50,8 @@
 			labelclass="control-label"
 		/>
 
-		<field name="description" type="textarea"
-			rows="3" cols="30" label="JGLOBAL_DESCRIPTION"
+		<field name="description" type="editor"
+               filter="raw" rows="3" cols="30" label="JGLOBAL_DESCRIPTION"
```
Comment (Fedik) [inline administrator/components/com_banners/models/forms/banner.xml]: use `raw` is a very bad idea, use `filter="JComponentHelper::filterText"` instead

**Comment by brianteeman** [comment]
I dont see the need to change this to a wyswiyg editor. As far as I can see this is an internal note only and is never displayed on the frontend so there is no need for formatting.

**Comment by JerryChr** [comment]
@brianteeman for my needs, an example could be where a client needs to show additional images/info together with the banner image in the ouput from the module, so I would use this field to do so in an override for the module - I could do that with the textarea as well, but the client can't read/code HTML so they would not know how to edit the content. I am sure there is other ways of doing this, but this seemed to be a small change that would give a bit more usefull possibilities.

**Comment by phproberto** [comment]
AFAIK description is not used anywhere (frontend or backend). It's only shown in the edit form. So this change could be a way to reuse it. We cannot add description to the current views, etc. for B/C reasons but I think changing the field type is ok. 

Nowadays it's quite normal having a banner with some kind of text shown there. If this allows to use it on overrides is ok for me.

`filter="JComponentHelper::filterText"` :+1:

**Comment by infograf768** [comment]
Also, not sure we need the Pagebreak and ReadMore buttons
```
buttons="true"
			hide="readmore,pagebreak"
```

**Comment by brianteeman** [comment]
I have tested this item :red_circle: unsuccessfully on d27f0122a7aa36f28d75c40cee7a348a70b33fd8<br /><br />The wysiwyg editor is loading but so are the pagebreak and readmore buttons<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/8704">issues.joomla.org/joomla-cms/8704</a>.</sub>

**Comment by infograf768** [comment]
@brianteeman 

Looks like the issue with hide buttons is not limited to this field.
I see it also for category.

Another regression I'm afraid...
Posted here: https://github.com/joomla/joomla-cms/issues/8712

**Comment by infograf768** [comment]
We now have a solution for the buttons

Please test and confirm https://issues.joomla.org/tracker/joomla-cms/8716

We could then confirm and RTC this one<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/8704">issues.joomla.org/joomla-cms/8704</a>.</sub>

**Comment by JerryChr** [comment]
@brianteeman after applying https://github.com/joomla/joomla-cms/pull/8716 it now hides the buttons. So to test this: Apply patch #8716 and #8704

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on db88d669e20ade860fa09b69ab5217f86909d248<br /><br />After applying #8716 this works<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/8704">issues.joomla.org/joomla-cms/8704</a>.</sub>

**Comment by infograf768** [comment]
I have tested this item :white_check_mark: successfully on db88d669e20ade860fa09b69ab5217f86909d248<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/8704">issues.joomla.org/joomla-cms/8704</a>.</sub>

**Comment by infograf768** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/8704">issues.joomla.org/joomla-cms/8704</a>.</sub>

**Pull request merged**